cookbooks/bcpc-hadoop/templates/default/hdp_hdfs-site.xml.erb

```diff
@@ -15,6 +15,11 @@
   </property>
 
   <property>
+    <name>dfs.datanode.balance.bandwidthPerSec</name>
```

Comment:
It looks like `ethtool` supports general hardware but `mii-tool` does not support all the NICs I could find.